src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/gc/shared/GenCollectedHeap.java

```diff
@@ -127,13 +127,11 @@ GenerationSpec spec(int level) {
     }
 
     if (level == 0) {
-      return (GenerationSpec)
-              VMObjectFactory.newObject(GenerationSpec.class,
-                      youngGenSpecField.getAddress());
+      return VMObjectFactory.newObject(GenerationSpec.class,
+              youngGenSpecField.getAddress());
     } else {
-      return (GenerationSpec)
-              VMObjectFactory.newObject(GenerationSpec.class,
-                      oldGenSpecField.getAddress());
+      return VMObjectFactory.newObject(GenerationSpec.class,
+              oldGenSpecField.getAddress());
```

Comment:
Maybe make each of these one line instead of two.

Reply:
Other lines in this file don't exceed 80 chars. So, I think it's better to preserver line break here.